src/main/java/core/basesyntax/controller/BetConsoleHandler.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.GenericDao;
+import core.basesyntax.lib.Inject;
+import core.basesyntax.model.Bet;
+import java.util.Scanner;
+
+public class BetConsoleHandler implements ConsoleHandler {
+    @Inject
+    GenericDao<Bet> betDao;
```

Comment:
generic types get erased at runtime
this field becomes just `GenericDao betDao`
as well as `GenericDao<User>` becomes `GenericDao`

let's not use `GenericDao` and create two separate interfaces instead - `BetDao` and `UserDao`

Reply:
and add access modifiers

Reply:
done